music21/romanText/tsvConverter.py

```diff
@@ -186,8 +187,15 @@ def __init__(self):
         self.changes = None
         self.phraseend = None
 
+        # the following attributes are overwritten by properties in TabChordV2
+        # because of changed column names in DCML v2
+        self.local_key = None
+        self.global_key = None
+        self.beat = None
+        self.measure = None
```

Comment:
can these be typed?  Basically anything that's being changed should be typed.

Reply:
It turns out that typing these causes mypy to object because they are overridden by properties in TabChordV2.
E.g., `music21/romanText/tsvConverter.py:425: error: Signature of "beat" incompatible with supertype "TabChordBase"`

This seems to be a bug in mypy, which there is an open issue about [here](https://github.com/python/mypy/issues/4125). (I like [this comment](https://github.com/python/mypy/issues/4125#issuecomment-870105336) on why overriding attributes by properties might be a good idea.) 

These aren't really "new" attributes, I just moved them from TabChordV1 to TabChordBase per your request. Is it alright if we just leave them untyped for now?

Reply:
ah -- if they're blocked by a bug in mypy, then we can leave them untyped.  If we need another round of review, maybe put a comment in the source code about why they're untyped.  Otherwise I'll forget and just ask again (or try to do it myself) next time the code is touched.